### PR TITLE
chore(website): improve mobile nav

### PR DESCRIPTION
**Pull request by TheSisb**
Cleans up the mobile nav a bunch and adds the theme switcher within it. I also refined the message in our responsive mobile view alert to be less verbose.

### CODE REVIEW

**Comment by TheSisb** [comment]
Blocked on Algolia's Docsearch.js for the time being.

**Review by SiTaggart — CHANGES_REQUESTED** [review]

File: packages/paste-website/src/components/ThemeSwitcher.tsx [hunk 1]
```diff
@@ -14,17 +15,21 @@ interface ThemeSwitcherProps {
...
     fontWeight: props.checked ? 'fontWeightBold' : 'fontWeightNormal',
     lineHeight: 'lineHeight30',
     paddingTop: 'space20',
     paddingBottom: 'space20',
     paddingRight: 'space40',
     paddingLeft: 'space40',
-    backgroundColor: props.checked ? 'colorBackgroundInverseLight' : 'transparent',
+    backgroundColor: props.checked
+      ? ['colorBackgroundPrimary', 'colorBackgroundPrimary', 'colorBackgroundInverseLight']
+      : ['colorBackgroundBody', 'colorBackgroundBody', 'transparent'],
```
Comment (SiTaggart) [inline packages/paste-website/src/components/ThemeSwitcher.tsx:30]: Whoa! This works and it's wild 🎉

**Review by richbachman — APPROVED** [review]
Outside of Simon's comments, this looks good to me.

File: packages/paste-website/src/components/site-wrapper/site-header/HamburgerToggle.tsx [hunk 2]
```diff
@@ -18,41 +20,40 @@ const Pattie: React.FC<BoxProps> = props => (
   />
 );
 
-const HamburgerToggle = React.forwardRef<HTMLButtonElement, HamburgerToggleProps>(({toggled}, ref) => {
-  const backgroundColor = toggled ? 'colorBackgroundBodyInverse' : 'colorBackgroundBody';
+const HamburgerToggle = React.forwardRef<HTMLButtonElement, HamburgerToggleProps>(({toggled, color}, ref) => {
   return (
-    <Box ref={ref} width="36px" height="26px" position="relative" transition="all .5s ease-in-out">
+    <Box ref={ref} width="36px" height="28px" position="relative" transition="all .5s ease-in-out">
```
[the remark this answers is not in the document]
Comment (TheSisb) [reply packages/paste-website/src/components/site-wrapper/site-header/HamburgerToggle.tsx]: Fixed.

**Pull request merged**